### PR TITLE
Add leap.nvim plugin

### PR DESCRIPTION
**Pull request by destag**
https://github.com/ggandor/leap.nvim

### CODE REVIEW

File: lua/nvcommunity/motion/leap/init.lua [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      { "gs", mode = { "n", "x", "o" }, desc = "Leap from windows" },
+    },
+    config = function(_, opts)
+      local leap = require "leap"
+      for k, v in pairs(opts) do
+        leap.opts[k] = v
+      end
+      leap.add_default_mappings(true)
+      vim.keymap.del({ "x", "o" }, "x")
+      vim.keymap.del({ "x", "o" }, "X")
```
Comment (lucario387) [inline lua/nvcommunity/motion/leap/init.lua]: Woah woah, i don't think this should be done

**Comment by xarthurx** [comment]
This one is good and a bit better than `hops` IMO.
Any plan to merge it?

**Comment by siduck** [comment]
> This one is good and a bit better than `hops` IMO. Any plan to merge it?

check the above comment

File: lua/nvcommunity/motion/leap/init.lua (hunk 1 above)
Comment (xarthurx) [reply lua/nvcommunity/motion/leap/init.lua]: @destag

Comment (destag) [reply lua/nvcommunity/motion/leap/init.lua]: Removed

**Review by siduck — APPROVED** [review]

**Pull request merged**